### PR TITLE
Add multi-drag example and swap example

### PR DESCRIPTION
**Pull request by andrie**
Adds example of using multi-drag for rank_list()

#54

### CODE REVIEW

File: R/sortable_options.R [hunk 1]
```diff
@@ -158,6 +164,7 @@ sortable_options <- function(
   handle = NULL,
   filter = NULL,
   draggable = NULL,
+  swap = NULL,
```
Comment (schloerke) [inline R/sortable_options.R:167]: Let's move this to the top. Let's add `multiDrag` as well.

Let's also move `...` to the beginning?

Comment (andrie) [reply R/sortable_options.R:167]: Agreed.

**Comment by schloerke** [comment]
Add a news item that `...` was moved to the front of `sortable_options()`

**Comment by schloerke** [comment]
Then LGTM!

**Pull request merged**